### DISCUSSION TITLE
RustDeskTranslation (Albanian)

### DISCUSSION THREAD

**Discussion by donald-2rm** [rustdesk/rustdesk#1943] [General]
We are a company located in Albania and since we have been recently using RustDesk we would like to contribute in translating it in our language, Albanian (Shqip).

If we have your consent in doing so and if you could give us some instructions that would be great.

**Comment by donald-2rm** [comment]
Any updates?

**Reply by rustdesk** [reply]
You need to ask nothing, please just go ahead.
Just like this https://github.com/rustdesk/rustdesk/pull/2153, he asked nothing before submit PR.

**Reply by donald-2rm** [reply]
https://github.com/rustdesk/rustdesk/pull/2397
